include/mbedtls/check_config.h

```diff
@@ -604,6 +605,11 @@
 #error "MBEDTLS_PSA_INJECT_ENTROPY is not compatible with actual entropy sources"
 #endif
 
+#if defined(MBEDTLS_PSA_INJECT_ENTROPY) &&              \
+    !defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG)
```

Comment:
I think you have inverted the logic here?

Reply:
Addressed in https://github.com/ARMmbed/mbedtls/pull/3895.